tutorials/02_installation.md

```diff
@@ -43,7 +43,7 @@ sudo apt install libignition-plugin-dev
 
 1. Install Ignition dependencies
   ```
-  sudo apt-get install libignition-cmake2-dev
+  sudo apt-get install libignition-cmake2-dev libignition-utils1-cli-dev
```

Comment:
The `packages.apt` file for this repo also has `libignition-tools-dev`, should that be added here? https://github.com/ignitionrobotics/ign-plugin/blob/7f3df50589c996b88aba9a64b7e15f7f7097e40b/.github/ci/packages.apt#L2

Reply:
17c2ce2